data-access/src/object_store/local.rs

```diff
@@ -185,7 +297,30 @@ pub fn local_unpartitioned_file(file: String) -> FileMeta {
             size: metadata.len(),
             path: file,
         },
-        last_modified: metadata.modified().map(chrono::DateTime::from).ok(),
+        last_modified: metadata.modified().map(DateTime::<Utc>::from).ok(),
+    }
+}
+
+struct LocalFileWriter {
+    path: String,
+}
+
+impl LocalFileWriter {
+    fn new(path: String) -> Result<Self> {
+        Ok(Self { path })
+    }
+}
+
+#[async_trait]
+impl ObjectWriter for LocalFileWriter {
+    async fn writer(&self) -> Result<Pin<Box<dyn AsyncWrite>>> {
+        let file = AsyncFile::open(&self.path).await?;
+        Ok(Box::pin(file))
+    }
+
+    fn sync_writer(&self) -> Result<Box<dyn Write + Send + Sync>> {
```

Comment:
FYI currently writing parquet requires Seek - but there is a suggestion here on how that could be removed https://github.com/apache/arrow-rs/issues/937